src/main/groovy/com/marklogic/gradle/task/roxy/scaffoldTemplates/SampleTestFiles.groovy

```diff
@@ -0,0 +1,158 @@
+   via the REST api in suite-setup then test them in your tests.
+8. Test data files should be placed in the test-data dir - this is important 
+   the framework will look in this folder. See the suite-setup.xqy for reading a test
+   data file from this folder.              
+
+*********v*********************************************************:)
+
+let \\\$uri := '/content/sample.xml'
+let \\\$expected-num-entries := '3'
+let \\\$expected-pid := 'PI78'
+let \\\$doc := fn:doc( \\\$uri )
+let \\\$num-entries :=  \\\$doc/manifest/zip-num-entries/text()
+let \\\$pid := \\\$doc/manifest/@productid/fn:string()
+return (
+  test:assert-true( \\\$num-entries = \\\$expected-num-entries,
+                  'For doc '|| \\\$uri || ' number of entries should be ' || \\\$expected-num-entries ||' but is ' || \\\$num-entries
+                   ),
+  test:assert-true( \\\$pid = \\\$expected-pid,
+                  'For doc '|| \\\$uri || ' expected product id is ' || \\\$expected-pid || ' but is ' || \\\$pid
+                   )
+)
+,
+xdmp:log('\$testName Tests ENDING.....')
+        """
+    }
+
+    static String getSetupXqy() {
+        return """
+xquery version '1.0-ml';
+
+import module namespace test='http://marklogic.com/roxy/test-helper' at '/test/test-helper.xqy';
+
+
+xdmp:log('\$testName Setup STARTING....')
+
+
+(: some stuff in there , each setup runs in its own txn :)   
+
+,
+
+xdmp:log('<% print testName %> Setup COMPLETE....')        """
+    }
+
+    static String getSuiteSetupXqy() {
+        return """
+xquery version '1.0-ml';
+
+import module namespace test='http://marklogic.com/roxy/test-helper' at '/test/test-helper.xqy';
+import module namespace c = 'http://marklogic.com/roxy/test-config' at '/test/test-config.xqy';
+import module namespace cfg = 'http://thieme.com/synone/config' at '/app/config/config.xqy';
+
+declare variable \\\$permissions := (xdmp:permission(\\\$cfg:SYNONE-READ, 'read'),
+                                  xdmp:permission(\\\$cfg:SYNONE-UPDATE, 'update') );
+
+xdmp:log('\$suiteName Suite Setup STARTING....')
+
+,
+
+try {(
+    (: insert sample in  db:)
+    let \\\$filenames := ('sample.xml')
+    for \\\$f in \\\$filenames
+    let \\\$uri := '/content/' || \\\$f
+    let \\\$file-to-load := test:get-test-file(\\\$f)
+    let \\\$_ := xdmp:log('\$suiteName Tests Suite Setup - loading '||\\\$uri)
+    return xdmp:document-insert(\\\$uri, \\\$file-to-load, \\\$permissions, 'manifests' )
+)} catch  (\\\$ex) {
+    xdmp:log(fn:concat('\$suiteName setup failed ', \\\$ex/error:format-string/text() ),'error')
+}
+
+,
+
+xdmp:log('\$suiteName Suite Setup COMPLETE....')
+        """
+    }
+
+    static String getSuiteTeardownXqy() {
+        return """
+xquery version '1.0-ml';
+
+import module namespace cfg = 'http://thieme.com/synone/config' at '/app/config/config.xqy';
+
+declare namespace error = 'http://marklogic.com/xdmp/error';
+
+xdmp:log('\$suiteName Suite Teardown STARTING....')
+
+,
+try {
```

Comment:
And same above as well, I think the log statements are fine to keep, but then just put a brief comment in there about why this module would be modified or deleted